docs/guide/testing_guide/unit_testing.rst

```diff
@@ -0,0 +1,25 @@
+.. _testing_guide:
+.. _unit_testing_guide:
+
+
+Getting the source code
+=======================
+
+getting ansible-pylibssh source::
+
+.. code-block:: shell-session
+    $ git clone https://github.com/ansible/pylibssh.git
+    $ cd pylibssh
+    $ pip install tox
+    $ tox -e build-dists
+
+Running tests
+==============
+
+.. code-block:: shell-session
+    $ tox -e test-binary-dists
```

Comment:
This should mention that it expects the test targets to be pre-built in `./dist/`